hawkbit-device-simulator/src/main/java/org/eclipse/hawkbit/simulator/amqp/DmfReceiverService.java

```diff
@@ -238,6 +295,7 @@ private void sendFeedback(final Long actionId, final AbstractSimulatedDevice dev
         case DOWNLOADED:
             spSenderService.sendActionStatusMessage(device.getTenant(), DmfActionStatus.DOWNLOADED,
                     device.getUpdateStatus().getStatusMessages(), actionId);
+            // TODO: Close Download-Only actions
```

Comment:
I think we can safely ignore removing the `actionId` from the `openActions`. Only drawback is that we have some trailing actionIds in the set for the download_only.